lib/tapjoy/autoscaling_bootstrap/configure_autoscaler.rb

```diff
@@ -16,6 +20,11 @@ def initialize(misc_config:, aws_env:, user_data:, **unused_values)
 
       private
 
+      # Check if ASG exists so we know whether to create or update
+      def autoscaling_group_exists
+        !Tapjoy::AutoscalingBootstrap::AWS::Autoscaling::Group.describe.nil?
```

Comment:
I ripped this from https://github.com/Tapjoy/tass/blob/3fd8f74d9cac05a59bc1c9a56bac9e1b695d1c2c/lib/tapjoy/autoscaling_bootstrap/autoscaling/group.rb#L46-L48, so I'll update it there as well.